### ISSUE TITLE
init does not start engine container

### ISSUE THREAD

**Issue by jfontan** [src-d/engine#120]
If engine container is already downloaded from a previous run and stopped it is not started. It also doesn't give any error.

```
$ srcd init
INFO[0000] starting daemon with working directory: /home/jfontan/tmp/borrar/empathy 

$ srcd web sql
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/src-d/engine/cmd/srcd/daemon.Client(0x553865, 0xa52860, 0xc0002d7080, 0xb2f52c)
	/home/travis/gopath/src/github.com/src-d/engine/cmd/srcd/daemon/daemon.go:63 +0x231
github.com/src-d/engine/cmd/srcd/cmd.startWebComponent.func1(0x10a6f40, 0x10cf330, 0x0, 0x0)
	/home/travis/gopath/src/github.com/src-d/engine/cmd/srcd/cmd/web.go:52 +0x61
github.com/src-d/engine/vendor/github.com/spf13/cobra.(*Command).execute(0x10a6f40, 0x10cf330, 0x0, 0x0, 0x10a6f40, 0x10cf330)
	/home/travis/gopath/src/github.com/src-d/engine/vendor/github.com/spf13/cobra/command.go:766 +0x2cc
github.com/src-d/engine/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x10a65c0, 0xb37049, 0x13, 0xc000270dc0)
	/home/travis/gopath/src/github.com/src-d/engine/vendor/github.com/spf13/cobra/command.go:852 +0x2fd
github.com/src-d/engine/vendor/github.com/spf13/cobra.(*Command).Execute(0x10a65c0, 0x407280, 0xc000088058)
	/home/travis/gopath/src/github.com/src-d/engine/vendor/github.com/spf13/cobra/command.go:800 +0x2b
github.com/src-d/engine/cmd/srcd/cmd.Execute()
	/home/travis/gopath/src/github.com/src-d/engine/cmd/srcd/cmd/root.go:41 +0x2d
main.main()
	/home/travis/gopath/src/github.com/src-d/engine/cmd/srcd/main.go:20 +0x20

$ srcd version
srcd cli version: 0.0.1
docker version: 1.38
srcd daemon version: not running
```

**Comment by smacker** [comment]
actually, I didn't test it with stopped engine container. Only with stopped components. Please let me know if it works for you or the issue still exists.

**Comment by jfontan** [comment]
It's fixed in `0.2.0`:

```
$ docker stop srcd-cli-daemon
srcd-cli-daemon

$ srcd init ~/go/src/github.com                      
INFO[0000] starting daemon with working directory: /home/jfontan/go/src/github.com 

$ docker ps                                          
CONTAINER ID        IMAGE                    COMMAND                  CREATED             STATUS              PORTS                    NAMES
cc88f6295402        srcd/cli-daemon:v0.2.0   "/srcd-server --work…"   10 seconds ago      Up 9 seconds        0.0.0.0:4242->4242/tcp   srcd-cli-daemon

$ srcd version                 
srcd cli version: v0.2.0
docker version: 1.38
srcd daemon version: 5252abd7377a5253135e53767fe247cc6400df8c
```
